pkg/installation/semver/version.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package semver is a wrapper for handling of semantic version values.
+package semver
+
+import (
+	"strings"
+
+	"github.com/pkg/errors"
+	k8sver "k8s.io/apimachinery/pkg/util/version"
+)
+
+// Version describes a semantic version value
+type Version k8sver.Version
+
+// String returns string representation of a semantic version
+// (https://semver.org) value with leading 'v' character.
+func (v Version) String() string {
+	vv := k8sver.Version(v)
+	s := (&vv).String()
+	if !strings.HasPrefix(s, "v") {
```

Comment:
This check is obsolete, as it will always evaluate to true, so that could be a one-liner.

Reply:
I have this because this pkg is meant to be a wrapper.
If we don't like the underlying k8sver package, we can change it without changing our code. Not all semver parsers are enforcing v*, but I want to.